src/randoop/JunitFileWriter.java

```diff
@@ -153,6 +143,19 @@ private File writeSubSuite(List<ExecutableSequence> sequencesForOneFile, String
     return file;
   }
 
+/*
+ * Writes a test method to the output stream for the sequence s with name "test"+testCounter. 
+ */
+  private void writeTest(PrintStream out, String className, int testCounter, ExecutableSequence s) {
+    out.println("  @Test");
+    out.println("  public void test" + testCounter + "() throws Throwable {");
```

Comment:
I haven't found anything that says why, but all JUnit4 examples are written as instance methods. The only reason it matters to us is the driver program generated by writeDriverFile, and that is a non-standard usage. I'm sticking with canonical JUnit4.